package/utils/spidev_test/Makefile

```diff
@@ -10,7 +10,6 @@ include $(INCLUDE_DIR)/kernel.mk
 
 PKG_NAME:=spidev-test
 PKG_RELEASE:=$(LINUX_VERSION)
-PKG_FLAGS:=nonshared
```

Comment:
Mhhh any idea why this was set as nonshared before?

Reply:
It was introduced by @chunkeey https://github.com/openwrt/openwrt/commit/bdaaf66e28bd45837e420ac2300df108091ec97a